modules/quant_purger/quant_purger.install

```diff
@@ -34,3 +34,23 @@ function quant_purger_schema() {
   ];
   return $schema;
 }
+
+/**
+ * Update configuration for module settings.
+ */
+function quant_purger_update_0001(&$sandbox) {
```

Comment:
I'm used to update hooks being like `1000`, `1001`, etc. Does having it use `0001` do something special?

Reply:
Nope, I just missed the convention when supporting multiple version of Drupal (it should use the lowest supported core version). As I understand them, the number doesn't really matter as long as its sequential, there are community guidelines on how to name them though.

> 1 or 2 digits for Drupal core compatibility (Drupal 8, 9, 10, etc.). This convention must be followed. If your module is compatible with multiple major versions (e.g., it has a core_version_requirement of '^8.8 || ^9'), use the lowest major core branch it is compatible with (8 in this example).

Reply:
I followed something like this one: 

```
mymodule_update_8201(): The first update for the custom or contributed module's 2.0.x versions, which are compatible with Drupal 8 and 9.
```

from https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Extension%21module.api.php/function/hook_update_N/9